app/src/test/java/org/mozilla/fenix/sync/ext/SyncedTabsListItemTest.kt

```diff
@@ -80,25 +80,24 @@ class SyncedTabsListItemTest {
     )
 
     @Test
-    fun `verify ordering of list items`() {
+    fun `WHEN given two devices THEN toComposeList generates two SyncedTabsListItem#DeviceSection`() {
```

Comment:
```suggestion
    fun `GIVEN two synced devices WHEN the compose list is generated THEN two device section is returned`() {
```

Stealing this comment from Christian https://github.com/mozilla-mobile/fenix/pull/23010#discussion_r802089174. 
"In general it's better to keep the method names out of the test names. Otherwise all test names will be stale if you anyone ever refactors those methods."